### PR TITLE
Phase3

### PR DESCRIPTION
**Pull request by Mottlly**

### CODE REVIEW

File: nextjs-blog/components/cardimage.js [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
-import Image from "next/image";
-import styles from "../styles/swipercontainer.module.scss";
+import Image from "next/legacy/image";
+import dbMocks from "../__mocks__/dbMocks.json";
 
 register();
 
-export default function CardImage({ imageUrl }) {
+export default function CardImage({
+  currentevent,
```
Comment (laurtann) [inline nextjs-blog/components/cardimage.js:9]: V minor, but convention is to camelCase

`currentevent -> currentEvent`
`setCurrentevent -> setCurrentEvent`

File: nextjs-blog/components/decisioncard.js [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
   return (
     <>
       <Head>
         <title>Decision Card Container</title>
       </Head>
       <div id="decisioncardcontainer" className={styles.CardAlpha}>
-        <TypingTextBox text="I Am Here..... And I can type myself!" />
-        <CardImage imageUrl="/pixelholder.jpg" />
+        <TypingTextBox text={dbMocks[currentevent].text} />
+        <CardImage
```
Comment (laurtann) [inline nextjs-blog/components/decisioncard.js:25]: Just for knowledge, it is possible to declare props and pass them into a child component in an object

File: nextjs-blog/components/speeddial.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import FullscreenOverlay from "../components/transoverlay";
+import SpeedDial from "@mui/material/SpeedDial";
+import SpeedDialIcon from "@mui/material/SpeedDialIcon";
```
Comment (laurtann) [inline nextjs-blog/components/speeddial.js:6]: Again, just for knowledge we could import these as follows

`import { SpeedDial, SpeedDialIcon } from "@mui/material"`

File: nextjs-blog/components/speeddial.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import SpeedDialAction from "@mui/material/SpeedDialAction";
+import PlayArrowIcon from "@mui/icons-material/PlayArrow";
+import CopyrightIcon from "@mui/icons-material/Copyright";
+import LocalMoviesIcon from "@mui/icons-material/LocalMovies";
```
Comment (laurtann) [inline nextjs-blog/components/speeddial.js:10]: Same here - we can rename to whatever we'd like. Can be helpful in some cases!
`import { LocalMovies as BananaIcon, Copyright as CopyrightIcon } from "@mui/icons-material"`

**Review by laurtann — APPROVED** [review]
Great work - left a few optional comments! Approved!

File: nextjs-blog/components/cardimage.js (hunk 1 above)
Comment (Mottlly) [reply nextjs-blog/components/cardimage.js:9]: Camel Case Corrected!

File: nextjs-blog/components/speeddial.js (hunk 3 above)
Comment (Mottlly) [reply nextjs-blog/components/speeddial.js:6]: Acknowledged and adjusted :D

**Pull request merged**